docs/Running-Tachyon-Fault-Tolerant-on-EC2.md

```diff
@@ -4,4 +4,149 @@ title: Running Tachyon with Fault Tolerance on EC2
+    $ export AWS_SECRET_ACCESS_KEY=<your secret access key>
+
+Next generate your EC2 [Key Pairs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html). Make sure to set the permissions of your private key file that only you can read it:
+
+    $ chmod 400 <your key pair>.pem
+
+Copy `deploy/vagrant/conf/ec2.yml.template` to `deploy/vagrant/conf/ec2.yml` by:
+
+    $ cp deploy/vagrant/conf/ec2.yml.template deploy/vagrant/conf/ec2.yml
+
+In the configuration file `deploy/vagrant/conf/ec2.yml`, set the value of `Keypair` to your keypair name and `Key_Path` to the path to the pem key.
+
+In the configuration file `deploy/vagrant/conf/tachyon.yml`, set the value of `Masters` to the number of TachyonMasters you want. In fault tolerant mode, value of `Masters` should be larger than 1.
+
+By default, the Vagrant script creates a [Security Group](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html) named *tachyon-vagrant-test* at [Region(**us-east-1**) and Availability Zone(**us-east-1a**)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html). The security group will be set up automatically in the region with all inbound/outbound network traffic opened. You can change the security group, region and availability zone in `ec2.yml`.
+
+Now you can launch the Tachyon cluster with Hadoop2.4.1 as under filesystem in us-east-1a by running the script under `deploy/vagrant`:
+
+    ./create <number of machines> aws
+
+Note that the `<number of machines>` above should be larger than or equal to `Masters` set in `deploy/vagrant/conf/tachyon.yml`.
+
+Each node of the cluster has a Tachyon worker, and each master node has a Tachyon master. The leader is in one of the master nodes.
+
+# Access the cluster
+
+**Access through Web UI**
+
+After command `./create <number of machines> aws` succeeds, you can see three green lines like below shown at the end of the shell output:
+
+    >>> Master public IP for Tachyon is xxx, visit xxx:19999 for Tachyon web UI<<<
+    >>> Master public IP for other softwares is xxx <<<
+    >>> visit default port of the web UI of what you deployed <<<
+
+The first line shows public IP for current leader of all Tachyon masters.
+
+The second line shows public IP for master of other softwares like Hadoop.
+
+Default port for Tachyon Web UI is **19999**.
+
+Default port for Hadoop Web UI is **50070**.
+
+Visit `http://{MASTER_IP}:{PORT}` in the browser to access the Web UIs.
+
+You can also monitor the instances state through [AWS web console](https://console.aws.amazon.com/console/home?region=us-east-1).
+
+**Access with ssh**
+
+The nodes set up are named to two groups. 
+
+One group contains `TachyonMaster`, `TachyonMaster2` and so on, representing all Tachyon masters, 
+one of them is leader, others are standbys, `TachyonMaster` is the master for other softwares like Hadoop. Each node also has workers for Tachyon
+and for other softwares like Hadoop.
```

Comment:
I see. I asked because the paragraph below also talks about "workers for Tachyon or Hadoop", so I wonder where the hadoop master node is. Or we may not care at all, but the description here might be a bit unclear.